### PR TITLE
Milestone 11: Added Client-side Form validation

### PR DESCRIPTION
**Pull request by prg-04**
###I have done the following to reach this milestone:

- [ ] Implemented client-side form validation
- [ ] Add custom error messages to make the page more user-friendly

### CODE REVIEW

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @prg-04, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

<div align='center'>

</div>

You have submitted a pull request with failing linter checks. Kindly fix the linter errors.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: validation.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  msgValid.innerHTML = '<i class="fa-sharp fa-solid fa-circle-check"></i>';
+  msgValid.style.color = 'green';
+  return true;
+}
+
+// Comment for the reviewer
+
+// I have used the variable in the html but am getting
+// no-unused-vars error for formValidate so i had to disable the eslint
+// eslint-disable-next-line no-unused-vars
```
Comment (divinecharlotte) [inline validation.js]: - [ ] kindly consider not disabling the eslint cause whatever might be the issue you are not allowed to disable eslint for this project consider fixing the issue you have locally before you push your codes

File: validation.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+  if (email.length === 0) {
+    emailErr.innerHTML = 'Email is required';
+    emailValid.innerHTML = null;
+
+    return false;
+  }
+  if (
+    !email.match(
+      /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/,
```
Comment (divinecharlotte) [inline validation.js]: - [ ] kindly consider adding email validation for only lowercase to meet the below requirement 
- > If a user submits the contact form with the email field containing uppercase characters an error message should show up.

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @Team , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive PR ✔️ 
- good commit messages 👍 

## Required Changes ♻️

- [ ] kindly consider adding a working link to your LinkedIn account under the author's section to make your readme look more professional
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by shella12 — APPROVED** [review]
Hi @prg-04 and @mj-e-boc , 


## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Linters are passing ✔️ 
- README is professional 👍 
- Implemented all requirements 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**